src/platformio.ini

```diff
@@ -140,7 +140,48 @@ upload_flags =
 src_filter = ${env_common_stm32.src_filter} -<rx_*.cpp>
 lib_deps =
 	https://github.com/PaoloP74/extEEPROM.git
+[env:Frsky_RX_R9SLIM_via_STLINK]
+platform = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.platform}
+board = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.board}
+build_unflags = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.build_unflags}
+build_flags = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.build_flags}
+board_build.ldscript = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.board_build.ldscript}
+src_filter = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.src_filter}
+upload_flags =
+    BOOTLOADER=bootloader/r9slim_no_btn_bootloader.bin
+    VECT_OFFSET=0x8000
+lib_deps = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.lib_deps}
+
+[env:Frsky_RX_R9SLIM_dump_via_STLINK]
```

Comment:
remove this target.

Reply:
sorry, I thought I removed them but turns out I didn't save before push